openapi.yaml

```diff
@@ -0,0 +1,675 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      summary: deprovision of a service binding
+      tags:
+        - ServiceBindings
+      operationId: serviceBinding.unbinding
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: id of the instance associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: id of the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+
+components:
+  parameters:
+    APIVersion:
+      name: X-Broker-API-Version
+      in: header
+      description: version number of the Service Broker API that the Platform will use
+      required: true
+      schema:
+        type: string
+      default: '2.13'
+
+    OriginatingIdentity:
+      name: X-Broker-API-Originating-Identity
+      in: header
+      description: identity of the user that initiated the request from the Platform
+      schema:
+        type: string
+
+  schemas:
+    Catalog:
+      type: object
+      properties:
+        services:
+          type: array
+          items:
+            $ref: '#/components/schemas/Service'
+
+    Service:
+      required:
+        - name
+        - id
+        - description
+        - bindable
+        - plans
+      properties:
+        name:
+          type: string
+        id:
+          type: string
+        description:
+          type: string
+        tags:
+          type: array
+          items:
+            type: string
+        requires:
+          type: array
+          items:
+            type: string
+            enum:
+              - syslog_drain
+              - route_forwarding
+              - volume_mount
+        bindable:
+          type: boolean
+        metadata:
```

Comment:
Is there an easy way to link to the [Service Metadata Conventions](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata) here? Maybe via a `description`?

Reply:
Not a bad idea. Added.